README.md

```diff
@@ -305,8 +306,26 @@ If your pull request (or issues) are not getting responses within a few days do
 by posting a comment such as "Any update on the status of this?".
 Sometimes GitHub notifications get lost.
 
-## Support
+## Support and citing
 
 Feel free to ask for help on the [Julia Discourse forum](https://discourse.julialang.org/),
 or in the `#natural-language` channel on julia-slack. (Which you can [join here](https://slackinvite.julialang.org/)).
 You can also raise issues in this repository to request improvements to the documentation.
+
+If you use this software as part of your research work, teaching or other activities, we would be grateful if you could cite the following:
+
+```
+@article{Kaushal3030,
+  doi = {10.21105/joss.01956},
+  url = {https://doi.org/10.21105/joss.01956},
+  year = {3030},
```

Comment:
This seems odd. 
The BibTeX was copied from https://joss.theoj.org/papers/10.21105/joss.01956